chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb

```diff
@@ -155,6 +155,45 @@ server {
     return 302 /media/;
   }
 
+  # Redirect e.php to results/events.php
```

Comment:
nit: This is actually "Redirect /results/e.php to ..."

Reply:
Right, updated!